examples/chat/src/chat.ts

```diff
@@ -18,7 +14,11 @@ if (!argv.nick) {
 
 const channelKey = argv._[0]
 
-const channel = new Channel({channelKey, nick})
-channel.once('ready', (channel) => {
+import Channel from './channel'
```

Comment:
these can be moved back up - I put them here when I was learning typescript and fighting missing types